pages/02.ndlc-program/program.md

```diff
@@ -13,5 +13,6 @@ visible: true
 
 ### Program proposals were accepted through December 15, 2015
 ### Poster session proposals being accepted through February 29, 2016
+#### [See Call for Poster Proposals pdf for more details](../../user/themes/bootstrapndlc/assets/poster_CFP.pdf)
```

Comment:
@emb03 let's not put the poster pdf in the theme, put it in `pages/02.ndlc-program/` instead